daemons/ipa-slapi-plugins/ipa-extdom-extop/ipa_extdom_common.c

```diff
@@ -271,7 +271,9 @@ int parse_request_data(struct berval *req_val, struct extdom_req **_req)
  *        sid (1),
  *        name (2),
  *        posix uid (3),
- *        posix gid (3)
+ *        posix gid (4),
```

Comment:
Could you please also add a comment that explains difference between V0/V1/V2 versions of the sequence. We need to have it documented somewhere. May be even add a document in `doc/designs/extdom-plugin.md`? Nothing formal, just explain what it does and how various versions of the control are structured.

Reply:
Added doc/designs/extdom-plugin-protocol.md. @sumit-bose since you are the original author, could you be so kind and check it?